contrib/script/podman_ps.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+logins() {
+    loginctl list-users | tail -n +2  | head -n -2 | grep -v gdm | awk '{print $2 }'
```

Comment:
I'd prefer `loginctl list-users --no-legend` over the tail/head. (I'd like a way to avoid the `awk`, too, but so far can't figure out how to do that without json/jq)

Reply:
I have simplified it and removed the grep -v gdm